packages/flutter/test/material/snack_bar_theme_test.dart

```diff
@@ -288,3 +298,12 @@ RawMaterialButton _getSnackBarButton(WidgetTester tester) {
     ).first,
   );
 }
+
+RenderParagraph _getTextRenderObject(WidgetTester tester, String text) {
```

Comment:
Looks like this specifically gets text render object of snack bars. Consider renaming to:

```suggestion
RenderParagraph _snackbarTextRenderObject(WidgetTester tester, String text) {
```

Reply:
Done